### PR TITLE
Use new and better canonical link to Google Groups

### PR DESCRIPTION
**Pull request by xuan-nguyen-swe**
https://github.com/etcd-io/website/issues/479

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ An initial agenda will be posted to the [shared Google docs][shared-meeting-note
 
 Meeting recordings are uploaded to official etcd [YouTube channel].
 
-Get calendar invitation by joining [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev) mailing group.
+Get calendar invitation by joining [etcd-dev](https://groups.google.com/g/etcd-dev) mailing group.
 
 Join Hangouts Meet: [meet.google.com/umg-nrxn-qvs](https://meet.google.com/umg-nrxn-qvs)
```
Comment (jmhbnz) [inline README.md:151]: We recently migrated to CNCF Zoom for meetings, would you mind please also updating this link to the zoom link specified on the website?  Refer: https://github.com/etcd-io/website/pull/714

**Review by jmhbnz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (xuan-nguyen-swe) [reply README.md:151]: sure, @jmhbnz

Comment (xuan-nguyen-swe) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -146,9 +146,9 @@ An initial agenda will be posted to the [shared Google docs][shared-meeting-note
 
 Meeting recordings are uploaded to official etcd [YouTube channel].
 
-Get calendar invitation by joining [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev) mailing group.
+Get calendar invitation by joining [etcd-dev](https://groups.google.com/g/etcd-dev) mailing group.
 
-Join Hangouts Meet: [meet.google.com/umg-nrxn-qvs](https://meet.google.com/umg-nrxn-qvs)
+Join CNCF-funded Zoom channel: [zoom.us/my/cncfetcdproject](https://zoom.us/my/cncfetcdproject)
 
 Join by phone: +1 405-792-0633‬ PIN: ‪299 906‬#
```
Comment (jmhbnz) [inline README.md:153]: Suggest we remove this phone number also, zoom provides audio dial in instructions through the normal flow when joining the meeting.

Comment (xuan-nguyen-swe) [reply README.md:153]: ah, ok. Didn't know that it's related to the virtual meeting room.

Comment (xuan-nguyen-swe) [reply README.md]: removed

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @xuan-nguyen-swe.

Nitpick - If you have time please squash the two commits relating to CNCF Zoom link and meeting phone number so those changes are a single commit.

**Comment by xuan-nguyen-swe** [comment]
> Nitpick - If you have time please squash the two commits relating to CNCF Zoom link and meeting phone number so those changes are a single commit.

sure, but I would like to do it tomorrow if that's ok for you.

**Review by ahrtr — APPROVED** [review]

**Comment by xuan-nguyen-swe** [comment]
> Nitpick - If you have time please squash the two commits relating to CNCF Zoom link and meeting phone number so those changes are a single commit.

Done.

**Review by serathius — APPROVED** [review]

**Comment by xuan-nguyen-swe** [comment]
Thank you all for your time to review and merge the PR!

**Pull request merged**